yarn.lock

```diff
@@ -11722,7 +11732,7 @@ react-lifecycles-compat@^3.0.4:
   resolved "https://registry.yarnpkg.com/react-lifecycles-compat/-/react-lifecycles-compat-3.0.4.tgz#4f1a273afdfc8f3488a8c516bfda78f872352362"
   integrity sha512-fBASbA6LnOU9dOU2eW7aQ8xmYBSXUIWr+UmF9b1efZBazGNO+rcXT/icdKnYm2pTwcRylVUYwW7H1PHfLekVzA==
 
-react-redux@^7.1.0:
+react-redux@^7.1.0, react-redux@^7.1.3:
```

Comment:
looks like two versions of react-redux are locked here, possibly redundant

Reply:
They're not being locked - this basically means two packages are requesting different ranges that can be resolved to the same version (i.e 7.1.3). I don't know why this updated though since I didn't touch anything - I think it's because common has a more recent dependency? I'll downgrade the version in common to match the others and remove this from the lockfile.